lib/rules/jsx-boolean-value.js

```diff
@@ -139,6 +161,7 @@ module.exports = {
         }
         if (
           isNever(configuration, exceptions, propName)
+          && configObject.assumeUndefinedIsFalse === false
```

Comment:
```suggestion
          && !configObject.assumeUndefinedIsFalse
```
with this change, you could probably just get rid of the entire `configObjectDefaultSetting` function?

Reply:
Yes, I removed configObjectDefaultSetting by applying the above code.